src/serviceWorker.js

```diff
@@ -141,4 +141,4 @@ export function unregister() {
                 console.error(error.message);
             });
     }
-}
+}
```

Comment:
Can we configure prettier to add new line at the end?

Reply:
I'll check this